test/rootchain/miscellaneous.js

```diff
@@ -1,7 +1,6 @@
 let RootChain = artifacts.require("RootChain");
 
 let { catchError, toHex } = require("../utilities.js");
-let { mineNBlocks } = require("./rootchain_helpers.js");
 
 contract('[RootChain] Miscellaneous', async (accounts) => {
```

Comment:
All these tests should be put in their appropriate files and delete this file

"Will not revert finalizeExit with an empty queue" can be in `transactions.js`

And the other 2 can be in blockSubmissions